src/web/GestureHandler.ts

```diff
@@ -479,7 +502,7 @@ function ensureConfig(config) {
   return props;
 }
 
-function asArray(value) {
+function asArray<T extends unknown>(value: T): T[] {
```

Comment:
Shouldn't this be in `utils.ts`?

Reply:
Probably, the web is still not finished.

Reply:
Actually, not really, this is used as a helper only there.